### DISCUSSION TITLE
重构应用主界面

### DISCUSSION THREAD

**Discussion by Jesse205** [aistra0528/Hail#199] [Ideas]
应用主界面的一些操作和UI不是很合理。
https://github.com/aistra0528/Hail/issues/135
如果要修改或者增加新的功能的话，会增加很多看不懂的无用代码。所以我提议要重构主界面。

## 重命名

首先就是首页的 adapter 名字从“按页面起名”改为“按功能起名”，也就是`HomeAdapter`改为`PagerAdapter`或者`TabAdapter`，`PagerAdapter`改为`AppsAdapter`

## 功能方法抽离

把原`PagerFragment`的长按菜单这些抽离出来，做成单个方法，降低代码理解难度以及方便以后复用这些代码。

@aistra0528

**Comment by aistra0528** [comment]
放手去做
